### PR TITLE
Fix for bug 98: Object doesn't support property or method 'substring'

### PR DESCRIPTION
**Pull request by madeleinefletcher**

### CODE REVIEW

File: JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ module Microsoft.ApplicationInsights.Telemetry.Common {
             if (value) {
                 value = Util.trim(value);
                 if (value.toString().length > DataSanitizer.MAX_STRING_LENGTH) {
-                    value = value.substring(0, DataSanitizer.MAX_STRING_LENGTH);
+                    value = value.toString().substring(0, DataSanitizer.MAX_STRING_LENGTH);
```
Comment (AlexBulankou) [inline JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts]: substring function takes start and *not-exclusive* end, so in this case it should actually be substring(0, DataSanitizer.MAX_STRING_LENGTH+1).
Overall I would rather switch it here and in other places to use substr, which behaves as you would expect - takes start_index and length.

**Comment by madeleinefletcher** [comment]
Looks like there is no difference between substr and substring, so I am going to leave it

File: JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts (hunk 1 above)
Comment (madeleinefletcher) [inline JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts]: Looks like there's no difference between substr and substring in the way they behave here, so I'm going to leave it.

Comment (AlexBulankou) [inline JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts]: sorry my bad, I cannot count

**Comment by AlexBulankou** [comment]
:shipit:

**Pull request merged**